servicetalk-grpc-netty/src/main/java/io/servicetalk/grpc/netty/DefaultGrpcServerBuilder.java

```diff
@@ -54,7 +54,7 @@
             .executionStrategy(defaultStrategy());
 
     DefaultGrpcServerBuilder(final HttpServerBuilder httpServerBuilder) {
-        this.httpServerBuilder = httpServerBuilder.protocols(h2Default());
+        this.httpServerBuilder = httpServerBuilder.protocols(h2Default()).allowDropRequestTrailers(true);
```

Comment:
We do not use methods with `boolean` argument on the builders. Instead, we provide methods that override the default. Examples:
1. `HttpServerBuilder#disableDrainingRequestPayloadBody()`
2. `H1SpecExceptions.Builder#allowPrematureClosureBeforePayloadBody()`
3. `DefaultAutoRetryStrategyProvider.Builder#disableWaitForLoadBalancer()`
Consider removing a boolean argument from `allowDropRequestTrailers` method for consistency, before we released it.

Reply:
Lets consider API changes as a followup PR. In general I'm not sure why we are averse to `boolean` argument methods, which seems like a canonical approach for builders. Passing the state as an arguments may remove additional conditional logic from user code:

```java
Builder builder = ...;
if(shouldAllow) {
  builder.shouldAllow();
} else {
  builder.noAllow();
} 
builder.build();
```

vs
```java
new Builder().shouldAllow(shouldAllow).build();
```

Reply:
We do not provide methods that flip the default back either. Only the one that changes the default:

```java
Builder builder = ...;
if(shouldAllow) {
  builder.shouldAllow();
}
builder.build();
```